### ISSUE TITLE
Generate WordPress event posts

### ISSUE THREAD

**Issue by oktalk** [dsmwebgeeks/webgeeksmailer#2]
We could probably generate a Wordpress event post at the same time we generate the MailChimp email.

**Comment by jplhomer** [comment]
YES! That would be so great. Probably just need to add an auth mechanism to the WP API.
